SmartDeviceLink/public/SDLSetGlobalProperties.h

```diff
@@ -69,7 +69,24 @@ NS_ASSUME_NONNULL_BEGIN
  @param menuLayout The layout of the top-level main menu
  @return The SetGlobalProperties RPC
  */
-- (instancetype)initWithHelpText:(nullable NSString *)helpText timeoutText:(nullable NSString *)timeoutText vrHelpTitle:(nullable NSString *)vrHelpTitle vrHelp:(nullable NSArray<SDLVRHelpItem *> *)vrHelp menuTitle:(nullable NSString *)menuTitle menuIcon:(nullable SDLImage *)menuIcon keyboardProperties:(nullable SDLKeyboardProperties *)keyboardProperties menuLayout:(nullable SDLMenuLayout)menuLayout;
+- (instancetype)initWithHelpText:(nullable NSString *)helpText timeoutText:(nullable NSString *)timeoutText vrHelpTitle:(nullable NSString *)vrHelpTitle vrHelp:(nullable NSArray<SDLVRHelpItem *> *)vrHelp menuTitle:(nullable NSString *)menuTitle menuIcon:(nullable SDLImage *)menuIcon keyboardProperties:(nullable SDLKeyboardProperties *)keyboardProperties menuLayout:(nullable SDLMenuLayout)menuLayout __deprecated_msg("Use initWithHelpText:timeoutText:vrHelpTitle:vrHelp:menuTitle:menuIcon:keyboardProperties:userLocation:menuLayout: instead");
+
+/**
+ Initialize SetGlobalProperties with all possible items
+
+ @param helpText A string that will be turned into TTS chunks for the help prompt
+ @param timeoutText A string that will be turned into TTS chunks for the timeout prompt
+ @param vrHelpTitle The title of the help interface prompt
+ @param vrHelp The items of the help interface prompt
+ @param menuTitle The title of the menu button
+ @param menuIcon The icon on the menu button
+ @param keyboardProperties The properties of a keyboard prompt
+ @param userLocation userLocation
+ @param menuLayout The layout of the top-level main menu
+ @return The SetGlobalProperties RPC
+ */
+
+- (instancetype)initWithHelpText:(nullable NSString *)helpText timeoutText:(nullable NSString *)timeoutText vrHelpTitle:(nullable NSString *)vrHelpTitle vrHelp:(nullable NSArray<SDLVRHelpItem *> *)vrHelp menuTitle:(nullable NSString *)menuTitle menuIcon:(nullable SDLImage *)menuIcon keyboardProperties:(nullable SDLKeyboardProperties *)keyboardProperties userLocation:(nullable SDLSeatLocation *)userLocation menuLayout:(nullable SDLMenuLayout)menuLayout;
```

Comment:
Instead of tweaking an existing init, just use the generated (i.e. use the RPC spec generator) init for `SDLSetGlobalProperties`.